README.md

```diff
@@ -143,6 +143,7 @@ For syntax, see [lookup.json](./lib/lookup.json), the available attributes are:
 "useGitClone": true          Use a shallow git clone instead of downloading the module
 "ignoreGitHead":             Ignore the gitHead field if it exists and fallback to using github tags
 "yarn":                      Install and test the project using yarn instead of npm
+"timeoutLength":             Number of milliseconds before timeout
```

Comment:
```suggestion
"testTimeout":             Number of milliseconds before timeout
```

Thinking about this more... If this is used only for tests, maybe `testTimeout` is a better name here?